optimum/intel/openvino/quantization.py

```diff
@@ -421,6 +430,14 @@ def _quantize_ovbasemodel(
                     for sub_model in sub_models:
                         _weight_only_quantization(sub_model.model, quantization_config)
                     self.model.clear_requests()
+                elif isinstance(self.model, OVModelForVisualCausalLM):
+                    language_model = self.model.language_model
+                    _weight_only_quantization(language_model.model, quantization_config, calibration_dataset)
+                    sub_model_names = ["vision_embeddings", "text_embeddings"] + self.model.additional_parts
+                    sub_models = [getattr(self.model, f"{name}_model") for name in sub_model_names]
+                    for sub_model in sub_models:
+                        _weight_only_quantization(sub_model, OVWeightQuantizationConfig(bits=8, sym=False))
```

Comment:
Why do you apply INT8_ASYM data-free quantization to the sub-models?
I'm just curious about the size of these models and whether you have observed any performance improvements.

Reply:
> Why do you apply INT8_ASYM data-free quantization to the sub-models?

This is what we've agreed on with @AlexKoff88 beforehand. 

> I'm just curious about the size of these models and whether you have observed any performance improvements.

For the VLMs in the description, text/vision embedding model sizes are in range of 150-500M parameters.

I've compared the inference speed for the two cases when we keep submodels in FP32 or compress to INT8_ASYM. Locally, I observe that the compressed scenario is about 5% slower. Maybe the situation is different on GPU devices.

But as I remember, @eaidova has mentioned, that on GPU there are some issues, when text/vision embeddings are inferred in INT8. So I'm inclined to keep them in FP32. 

Another option is to introduce some parameter threshold. For example, by default we compress only 1B+ LLMs.

@AlexKoff88 What do you think?

Reply:
WIll be done in the follow-up PR.